tools/build_variables.bzl

```diff
@@ -373,6 +373,7 @@ libtorch_extra_sources = libtorch_core_jit_sources + [
     "torch/csrc/jit/mobile/module.cpp",
     "torch/csrc/jit/mobile/observer.cpp",
     "torch/csrc/jit/mobile/optim/sgd.cpp",
+    "torch/csrc/jit/mobile/runtime_bytecode_version.cpp",
```

Comment:
Could you rename this to something like runtime_compatibility.cpp. I'm gonna do a similar thing to the model_bytecote.cpp when I add the model ops api. I think the seperation between model and runtime makes sense, but it also makes sense to me to put apis that will be used together as close together as possible